tst/EnergyPlus/unit/CurveManager.unit.cc

```diff
@@ -243,7 +243,622 @@ TEST_F( EnergyPlusFixture, Tables_TwoIndVar_Malformed ) {
+        "Dimensionless,           !- Input Unit Type for Y",
+        "Dimensionless,           !- Output Unit Type",
+        ",                        !- Normalization Reference",
+        "0,                       !- X Value #1",
+        "1,                       !- Y Value #1",
+        "0,                       !- Output Value #1",
+        "1,                       !- X Value #2",
+        "1,                       !- Y Value #2",
+        "2,                       !- Output Value #2",
+        "2,                       !- X Value #3",
+        "1,                       !- Y Value #3",
+        "2,                       !- Output Value #3",
+        "3,                       !- X Value #4",
+        "1,                       !- Y Value #4",
+        "3,                       !- Output Value #4",
+        "4,                       !- X Value #5",
+        "1,                       !- Y Value #5",
+        "4,                       !- Output Value #5",
+        "5,                       !- X Value #6",
+        "1,                       !- Y Value #6",
+        "2,                       !- Output Value #6",
+        "0,                       !- X Value #7",
+        "2,                       !- Y Value #7",
+        "0,                       !- Output Value #7",
+        "1,                       !- X Value #8",
+        "2,                       !- Y Value #8",
+        "2,                       !- Output Value #8",
+        "2,                       !- X Value #9",
+        "2,                       !- Y Value #9",
+        "2,                       !- Output Value #9",
+        "3,                       !- X Value #10",
+        "2,                       !- Y Value #10",
+        "3,                       !- Output Value #10",
+        "4,                       !- X Value #11",
+        "2,                       !- Y Value #11",
+        "4,                       !- Output Value #11",
+        "5,                       !- X Value #12",
+        "2,                       !- Y Value #12",
+        "2;                       !- Output Value #12"  });
+
+    ASSERT_FALSE(process_idf(idf_objects));
+
+    EXPECT_EQ(0, CurveManager::NumCurves);
+    CurveManager::GetCurveInput();
+    CurveManager::GetCurvesInputFlag = false;
+    ASSERT_EQ(2, CurveManager::NumCurves);
+
+    // QuadraticLinear curve type
+    EXPECT_EQ("QUADRATICLINEAR", CurveManager::GetCurveType(1));
+    EXPECT_EQ("TESTTABLEMINMAX", CurveManager::GetCurveName(1));
+    EXPECT_EQ(1, CurveManager::GetCurveIndex("TESTTABLEMINMAX"));
+    bool error = false;
+    int index = CurveManager::GetCurveCheck("TESTTABLEMINMAX", error, "TEST");
+    EXPECT_FALSE(error);
+    EXPECT_EQ(1, index);
+    Real64 min1, max1, min2, max2;
+    CurveManager::GetCurveMinMaxValues(1, min1, max1, min2, max2);
+    EXPECT_EQ(-1.0, min1); // user entered value is retained, however, CurveValue will test against lower array boundary and use aray minimum
```

Comment:
@jasondegraw For types like `LinearInterpolationOfTable` where we don't allow extrapolation, then I would have expected any user input limits that are beyond that to be warned and overwritten during getinput.  So, the original user limits should be gone by here.  But I may be missing something.